### ISSUE TITLE
Nice Site, But where is the headless cms hosted and how?

### ISSUE THREAD

**Issue by animemandir** [afgonullu/personal-website-v1#1]
I suppose you have deployed a headless ghost cms on heroku, Can you share the link of the heroku deployment?

**Comment by afgonullu** [comment]
That is right. Headless CMS is hosted on Heroku. Unfortunately since there is an accessible Next.js site for that, ghost UI is on private mode and cannot be accessible and also doesn't have any purpose anyway. I can't share the admin dashboard, I am sure you would understand. :) 

Also there is nothing to be seen there. It is pretty basic ghost setup. I used this on heroku -> https://github.com/afgonullu/ghost-on-heroku
